### ISSUE TITLE
beginner_source/examples_autograd/polynomial_autograd.py 오타수정

### ISSUE THREAD

**Issue by hjvision96** [PyTorchKorea/tutorials-kr#495]
## 문서 URL
_수정이 필요한 URL을 남겨주세요. (예. https://tutorials.pytorch.kr/beginner/basics/intro.html)_
- **URL**: https://github.com/PyTorchKorea/tutorials-kr/blob/master/beginner_source/examples_autograd/polynomial_autograd.py

## 변경 사항
_(1)어떤 단어 / 문장 / 내용이 (2)어떻게 변경되어야 한다고 생각하세요?_
경사하강법(gradient descent)를 -> 경사하강법(gradient descent)을

## 추가 정보
_위와 같이 생각하신 이유 또는 다른 참고할 정보가 있다면 알려주세요._
단어의 마지막 글자의 받침과 그 뒤에 이어지는 조사의 형태를 맞추어, 읽는 데 자연스러움을 더할 수 있어 수정을 요청하게 되었습니다.
